qa/tasks/mgr/dashboard/test_api.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from .helper import DashboardTestCase
+from . import DEFAULT_VERSION
+
+import unittest
+
+
+class VersionReqTest(DashboardTestCase, unittest.TestCase):
+    def test_version(self):
+        for (version, expected_status) in [
+              (DEFAULT_VERSION, 200),
+              (None, 415),
+              ("99.99", 415)
+            ]:
+            with self.subTest(version=version):
+                 self._get('/api/summary', version=version)
+                 self.assertStatus(expected_status)
```

Comment:
Given that all this logic is internal to CherryPy, I think it's enough with the unit tests you added. What do you think @alfonsomthd ?